error-handlers/index.js

```diff
@@ -0,0 +1,3 @@
+exports.handleCustomErrors = require('./handleCustomErrors')
```

Comment:
This could probably be neatened up a little so you are exporting as an object with the 3 function in rather than 3 nested objects. Other than that, good work